components/base/InlineCTA.tsx

```diff
@@ -54,17 +58,22 @@ const ArrowStyle = styled.div`
 type Props = {
   icon: string;
   text: string;
+  href: string;
```

Comment:
Will we ever use this component for external links (that would open in a new tab)? For other link components like the Card, I've been using `link?: { href?: string; external?: boolean };` as a prop to allow for that

It's totally fine if we're not planning on using this particular component like that, but I wanted to make sure we considered it!

Reply:
Good question! We have an external CTA component that has a different design (perhaps to distinguish it between external and internal? not sure 🤷 ) but I wouldn't be against accounting for both external and internal linking via the Inline CTA.

Reply:
Oh that's a good point! I forgot we had another component for that specifically. Thanks for reminding me 🙂

Reply:
Thanks for the APPROVED. As an aside, if in the future we do decide to use this CTA for an external link, we can always add that option!